frontend/src/index.pug

```diff
@@ -19,6 +19,7 @@ html
     script(src="https://cdn.jsdelivr.net/npm/jszip@3.1/dist/jszip.min.js")
 
     script(src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@9.14/build/highlight.min.js")
+    script(src="https://cdn.jsdelivr.net/npm/seedrandom@3.0.5/seedrandom.min.js")
```

Comment:
possible to group the scripts from same directory together? e.g. npm, gh

Reply:
sure.

Reply:
done.